coding-standards/php.md

```diff
@@ -11,5 +11,24 @@ https://www.php-fig.org/psr/psr-12/
 [https://cheatsheetseries.owasp.org/cheatsheets/Laravel_Cheat_Sheet.html](https://cheatsheetseries.owasp.org/cheatsheets/PHP_Configuration_Cheat_Sheet.html)
 
 ## Enforcing tools and config
```

Comment:
https://github.com/OsmosysSoftware/dev-standards/blob/main/coding-standards/dotnet.md

I would recommend to have similar heading for similar sections atleast which we have for .NET